README.md

```diff
@@ -10,6 +10,9 @@ at the heart of the [Wikibase software](http://wikiba.se/).
 
 ### 4.0.0 (dev)
 
+* Made the library a pure JavaScript library.
+* Removed MediaWiki ResourceLoader module definitions.
+* Raised DataValues JS library version requirement to 0.10.0.
```

Comment:
I already fixed the minor concerns I had with this in 161970d1a02a7855e3ba6b1e136a267e5068f0e4.